src/server/util/db.js

```diff
@@ -22,5 +22,5 @@ exports.usingTransaction = async (client, method) => {
     }
 };
 
-exports.withClientTransacation = method =>
+exports.withClientTransaction = method =>
```

Comment:
heh -- was it used anywhere else? 😳

Reply:
Oh good call! It's in the roles db. I'll do a quick PR to fix that before merging this one in.